src/OpenLoco/Objects/CargoObject.cpp

```diff
@@ -0,0 +1,39 @@
+#include "CargoObject.h"
+#include "../Interop/Interop.hpp"
+
+namespace OpenLoco
+{
+    // 0x0042F533
+    bool CargoObject::validate() const
+    {
+        if (var_2 > 3840)
```

Comment:
`3840` magic number, can we make a constant for it?

Reply:
until i know what var_2 represents its not going to be of much use.

Reply:
good point